README.md

```diff
@@ -109,6 +109,12 @@ For MSAL version 1.1.14 and above, distribution of MSAL binary framework as a Sw
     1. Rules -> Branch : master (For latest MSAL release)
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->